src/tests/daos_perf.c

```diff
@@ -1278,7 +1278,12 @@ main(int argc, char **argv)
 			ts_zero_copy = true;
 			break;
 		case 'f':
-			strncpy(ts_pmem_file, optarg, PATH_MAX - 1);
+			if (strlen(optarg) > PATH_MAX - 5) {
```

Comment:
this should probably be strnlen(...with PATH_MAX) since it's a user option.  I think you'll get a coverity hit here.

Reply:
Fixed and pushed a change